### PR TITLE
adding new tooling articles to the learning area sidebar

### PR DESCRIPTION
**Pull request by chrisdavidmills**
This PR adds the following to the Learning Area sidebar:

* https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/GitHub
* https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Understanding_client-side_tools, and all of its sub-srticles

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
I've tested this on my local Kuma install and it looks to be working fine. @Elchi3 , can you give it a quick verification?

File: macros/LearnSidebar.ejs [hunk 1]
```diff
@@ -1724,6 +1780,20 @@ var text = mdn.localStringMap({
         </ol>
     </details>
   </li>
+  <li><a href="<%=baseURL%>Tools_and_testing/GitHub"><%=text['Git_and_GitHub']%></a></li>
```
Comment (Elchi3) [inline macros/LearnSidebar.ejs:1783]: It works for me, too, but the "Git and GitHub" link is now the only one without sub pages and thus it looks a bit lost to me.

learnsidebar

Comment (chrisdavidmills) [reply macros/LearnSidebar.ejs:1783]: Yeah, I agree with you, but I wasn't sure what to do about that. I mean, it is a page at the module level, but it doesn't link to articles on our site — it links to external GitHub articles.

Maybe I could just remove it.

Comment (Elchi3) [reply macros/LearnSidebar.ejs:1783]: Or you add the guides listed here? http://wiki.localhost.org:8000/en-US/docs/Learn/Tools_and_testing/GitHub#Guides

Comment (chrisdavidmills) [reply macros/LearnSidebar.ejs:1790]: OK, I've just tried adding them. I've not added in language strings for the external articles, as they are not translated. I'll try testing the update now.

Comment (chrisdavidmills) [reply macros/LearnSidebar.ejs:1790]: Dammnit, I can't seem to test it. I pulled the changes to my local kumascript submodule, then restarted docker, but it won't show me the updates — it just shows me the original sidebar update as it was at the start of this PR....

File: macros/LearnSidebar.ejs [hunk 2]
```diff
@@ -1724,7 +1787,33 @@ var text = mdn.localStringMap({
         </ol>
     </details>
   </li>
-
+  <li><a href="<%=baseURL%>Tools_and_testing/GitHub"><%=text['Git_and_GitHub']%></a></li>
```
Comment (Elchi3) [inline macros/LearnSidebar.ejs:1790]: I tested it and it works! Please remove this first list element though:

```suggestion
```

Comment (chrisdavidmills) [reply macros/LearnSidebar.ejs]: Oops; I left that in by mistake! Removed now.

**Pull request merged**